modules/distributor/receiver/metrics.go

```diff
@@ -0,0 +1,92 @@
+package receiver
```

Comment:
Should we indicate where this comes from or import it in a different way? I did it this way b/c we only need some bits from the entire pkg and all this wasn't in a single file

Reply:
If we just copied this code from somewhere else then we need to check the license of the repo it came from. If the license requires attribution then, yeah, we should link back to where it was copied from.

Reply:
Good point. Added the same license entry and specified where the code codes from.